### PR TITLE
New Disabled touch on MaterialDayPicker in DetailsFragment.

### LINKED ISSUE #17: Disable touch on MaterialDayPicker in DetailsFragment
**Describe the bug**
A user can click on the MaterialDayPicker shown in the details fragment and can select the days which were not selected initially or unselect the days which they selected.

**To Reproduce**
1. Open Quiet Hours
2. Create a profile.
3. Click on the profile.
4. Click on the MaterialDayPicker shown in the profile.

**Expected behavior**
The user should not be able to change the value of MaterialDayPicker in details fragment.

**Would you like to work on the issue?**

- [ ] Yes
- [ ] No
- Other: Open for all

### PR DESCRIPTION
**Pull request by JanuKShah**
Fixes #17 

Changes: Final changes for adding a hacky view.

### CODE REVIEW

File: app/src/main/res/layout/fragment_details.xml [hunk 1]
```diff
@@ -22,10 +22,8 @@
                 android:layout_width="wrap_content"
                 android:layout_height="wrap_content"
                 android:layout_below="@id/txt1"
-                android:layout_marginTop="20dp"
-                android:hint="Start Time"
-
                 android:layout_alignStart="@id/end_txt"
+                android:layout_marginTop="50dp"
```
Comment (rob729) [inline app/src/main/res/layout/fragment_details.xml:29]: Reason for these changes?

Comment (JanuKShah) [reply app/src/main/res/layout/fragment_details.xml:29]: yes. If I keep it 50 then only it is properly displayed

File: app/src/main/res/layout/fragment_details.xml [hunk 2]
```diff
@@ -102,13 +102,20 @@
...
                 android:layout_width="match_parent"
                 android:layout_height="wrap_content"
-                android:layout_below="@id/col2"
                 android:layout_marginStart="20dp"
-                android:layout_marginTop="40dp"
-                android:layout_marginEnd="20dp"
-                android:clickable="false"
-                android:filterTouchesWhenObscured="true" />
+		            android:layout_marginEnd="20dp"
+            		android:layout_marginTop="40dp"
```
Comment (rob729) [inline app/src/main/res/layout/fragment_details.xml]: Indentation issue. Fix it.

**Comment by rob729** [comment]
@JanuKShah 
First of all, you need to merge the master code properly, and in the case of merge conflicts need to solve it properly.
The audio mode image view has nothing to do with this PR then why is it being shown as an addition.

**Comment by JanuKShah** [comment]
> @JanuKShah
> First of all, you need to merge the master code properly, and in the case of merge conflicts need to solve it properly.
> The audio mode image view has nothing to do with this PR then why is it being shown as an addition.

Since I had to reset my previous commits some  commits were coming in between so they  were undone. That is why it is showing up as an addition.

**Comment by rob729** [comment]
It is coming because either commits are not squashed properly or merge conflict not resolved properly. Have a look again at the PR.

**Comment by rob729** [comment]
@JanuKShah Everything looks fine. Just squash your commits.

**Comment by rob729** [comment]
@JanuKShah Any plans of squashing the commits?

**Comment by JanuKShah** [comment]
> @JanuKShah Any plans of squashing the commits?

I am searching but it is not working since I resolved the issues on github itself

**Comment by rob729** [comment]
Pull the changes to your system and then try to make the changes.
I want #17 to be resolved as soon as possible because it a bug and not a feature request which can wait for a long time.

**Pull request closed without merging**